package.json

```diff
@@ -37,6 +37,8 @@
   "dependencies": {
     "semver": "^5.4.1",
     "uuid": "^3.1.0",
+    "metarhia-common": "0.0.19",
```

Comment:
@nechaido both `0.0.19` and `^0.0.19` mean the same thing. Caret ranges only allow updates that don't modify the leftmost non-zero part of the version.